src/ServiceFabric/ServiceFabric/Template/Windows/template.json

```diff
@@ -151,14 +151,22 @@
     },
     "vmNodeType0Size": {
       "type": "string",
-      "defaultValue": "Standard_D2_V2"
+      "defaultValue": "Standard_D2s_v3"
     },
     "durabilityLevel": {
       "type": "string",
-      "defaultValue": "Bronze"
+      "allowedValues": [
+        "Silver",
+        "Gold"
+      ],
+      "defaultValue": "Silver"
     },
     "reliabilityLevel": {
       "type": "string",
+      "allowedValues": [
```

Comment:
might need to allow for Platinum here if any of the sql team uses these or equivalent